tests/vasp/test_vasp_import.py

```diff
@@ -46,9 +47,10 @@ def test_import(self):
             self.file_location, "../static/vasp_test_files/full_job_minor_glitch"
         )
         with warnings.catch_warnings(record=True) as w:
-            warnings.simplefilter("always")
+            warnings.simplefilter("ignore")
+            warnings.simplefilter("always", category=VasprunWarning)
             self.project.import_from_path(path=folder_path, recursive=False)
-            self.assertEqual(len(w), 4)
+            self.assertEqual(len(w), 3)
```

Comment:
This was bad test design on my part! Perhaps a better way to check the errors is:
```suggestion
            self.assertTrue(all([isinstance(warn, VasprunWarning) for warn in w])) 
```

Reply:
With the patch from Marvin, this would always be `True`. Thus, checking for exactly 3 `VasprunWarnings` sounds reasonable to me.
I will now merge this PR and release a new `pyiron_atomistics`. Feel free to adapt the test in another PR, though.